### PR TITLE
Add Finnish (fi)

### PR DESCRIPTION
**Pull request by samip5**

### CODE REVIEW

**Comment by samip5** [comment]
So it absolutely needs to have test cases for all of the different ones?

**Comment by balloob** [comment]
@samip5 Yes. All sentence templates need to have at least 1 test sentence

File: languages.yaml [hunk 1]
```diff
@@ -31,12 +31,14 @@ el:
 en:
   nativeName: English
 es:
-  nativeName: Español
+  nativeName: "Espa\xF1ol"
```
Comment (balloob) [inline languages.yaml:34]:
```suggestion
  nativeName: "Español"
```

File: sentences/fi/_common.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    area: "{area}"
+    what_is: "(mikä on | mikä se on | mitä varten se on | mikä)"
+    is_it: "(onko | onko se)"
+    brightness: "{brightness} [percent]"
+    turn_on: "(pane | päälle | käynnistä)"
+    turn_off: "(sammuta | sammuttaa | lopeta)"
+    temp: "(lämpötila)"
+    temperature: "{temperature} [astetta] [{temperature_unit}]"
+  skip_words:
+    - "Ole kiltti"
```
Comment (balloob) [inline sentences/fi/_common.yaml:61]: Indentation is wrong here. Makes expansion rules and skip words part of the lists.
```suggestion
expansion_rules:
  name: "{name}"
  area: "{area}"
  what_is: "(mikä on | mikä se on | mitä varten se on | mikä)"
  is_it: "(onko | onko se)"
  brightness: "{brightness} [percent]"
  turn_on: "(pane | päälle | käynnistä)"
  turn_off: "(sammuta | sammuttaa | lopeta)"
  temp: "(lämpötila)"
  temperature: "{temperature} [astetta] [{temperature_unit}]"
skip_words:
  - "Ole kiltti"
```

**Comment by balloob** [comment]
Looks like another Finnish PR #599 has been merged. Going to close this PR. You can open a new contribution that merges your work with the existing work.

**Pull request closed without merging**